From 908963b31e7a628a3cd24077b89839c7ff5fbda7 Mon Sep 17 00:00:00 2001
From: Termineitor244 <termineitor244@hotmail.com>
Date: Sun, 18 Aug 2024 09:14:35 -0600
Subject: [PATCH 054/125] Some nested recipe fixes (#75701)

* Nested aztec stone weapon recipes

* Kevlar Jumpsuit nested recipes fix

* Apply fix
---
 data/json/recipes/nested.json         | 12 ++++++------
 data/json/recipes/weapon/bashing.json |  8 ++++----
 2 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/data/json/recipes/nested.json b/data/json/recipes/nested.json
index ef251c3d51..821be6b5c0 100644
--- a/data/json/recipes/nested.json
+++ b/data/json/recipes/nested.json
@@ -2422,7 +2422,7 @@
     "subcategory": "CSC_*_NESTED",
     "name": "standard Kevlar jumpsuits",
     "description": "Recipes related to constructing custom-made jumpsuits meant to keep the wearer safe from harm.",
-    "skill_used": "fabrication",
+    "skill_used": "tailor",
     "nested_category_data": [ "survivor_jumpsuit", "xssurvivor_jumpsuit", "xlsurvivor_jumpsuit" ],
     "difficulty": 6
   },
@@ -2434,7 +2434,7 @@
     "subcategory": "CSC_*_NESTED",
     "name": "light Kevlar jumpsuits",
     "description": "Recipes related to constructing custom-made jumpsuits meant to keep the wearer safe from harm while still being flexible.",
-    "skill_used": "fabrication",
+    "skill_used": "tailor",
     "nested_category_data": [ "lsurvivor_jumpsuit", "xs_lsurvivor_jumpsuit", "xl_lsurvivor_jumpsuit" ],
     "difficulty": 5
   },
@@ -2446,7 +2446,7 @@
     "subcategory": "CSC_*_NESTED",
     "name": "steel-plated Kevlar jumpsuits",
     "description": "Recipes related to constructing custom-made jumpsuits meant to keep the wearer REALLY safe from harm, although they are a bit heavy.",
-    "skill_used": "fabrication",
+    "skill_used": "tailor",
     "nested_category_data": [ "hsurvivor_jumpsuit", "xs_hsurvivor_jumpsuit", "xl_hsurvivor_jumpsuit" ],
     "difficulty": 7
   },
@@ -2458,7 +2458,7 @@
     "subcategory": "CSC_*_NESTED",
     "name": "Kevlar firesuits",
     "description": "Recipes related to constructing custom-made jumpsuits meant to keep the wearer safe from harm and intense fires.",
-    "skill_used": "fabrication",
+    "skill_used": "tailor",
     "nested_category_data": [ "fsurvivor_jumpsuit", "xs_fsurvivor_jumpsuit", "xl_fsurvivor_jumpsuit" ],
     "difficulty": 8
   },
@@ -2470,7 +2470,7 @@
     "subcategory": "CSC_*_NESTED",
     "name": "winter Kevlar jumpsuits",
     "description": "Recipes related to constructing custom-made jumpsuits meant to keep the wearer safe from harm and freezing cold temperatures.",
-    "skill_used": "fabrication",
+    "skill_used": "tailor",
     "nested_category_data": [
       "wsurvivor_jumpsuit_nofur",
       "xs_wsurvivor_jumpsuit_nofur",
@@ -2489,7 +2489,7 @@
     "subcategory": "CSC_*_NESTED",
     "name": "Kevlar wetsuits",
     "description": "Recipes related to constructing custom-made jumpsuits meant to keep the wearer safe from harm and keep water out.",
-    "skill_used": "fabrication",
+    "skill_used": "tailor",
     "nested_category_data": [
       "h20survivor_jumpsuit",
       "xs_h20survivor_jumpsuit",
diff --git a/data/json/recipes/weapon/bashing.json b/data/json/recipes/weapon/bashing.json
index 135a0b6330..b18ca8d356 100644
--- a/data/json/recipes/weapon/bashing.json
+++ b/data/json/recipes/weapon/bashing.json
@@ -329,8 +329,8 @@
     "type": "recipe",
     "activity_level": "LIGHT_EXERCISE",
     "result": "aztec_sword_stone",
-    "category": "CC_WEAPON",
-    "subcategory": "CSC_WEAPON_CUTTING",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
     "skill_used": "fabrication",
     "difficulty": 1,
     "time": "10 m",
@@ -555,8 +555,8 @@
     "type": "recipe",
     "activity_level": "LIGHT_EXERCISE",
     "result": "aztec_spear_stone",
-    "category": "CC_WEAPON",
-    "subcategory": "CSC_WEAPON_CUTTING",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
     "skill_used": "fabrication",
     "difficulty": 1,
     "time": "10 m",
-- 
2.45.2

